src/fetchData.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-redeclare */
+/* global, fetch document */
+
+import loadDOM from "./loadDOM";
+
+const fetchData = () => {
+  loadDOM();
+  let location = document.getElementById("city");
+  let city = location.value;
+  let fetchUrl = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=fa1c7f10e6e75b7dc652e684202b5bd1`;
+  let displayLocation = document.querySelector(".location");
+
+  fetch(fetchUrl, { mode: "cors" })
+    .then(response => {
+      return response.json();
+    })
+    .then(response => {
+      const cityName = response.name;
+      const { temp, humidity, pressure } = response.main;
+      const { deg } = response.wind;
+      const { country } = response.sys;
+
+      displayLocation.textContent = `${cityName}, ${country}`;
+
+      let loadTemp = document.querySelector(".loadTemp");
+      let fahren = Math.round((temp * 9) / (5 + 32));
+      let celsius = Math.round(((fahren - 32) * 5) / 9);
+      let statusF = document.getElementById("f");
+      let statusC = document.getElementById("c");
+      let loadLetter = document.querySelector(".loadLetter");
+      let loadHumidity = document.querySelector(".humidity");
+      let loadPressure = document.querySelector(".pressure");
+      let loadWind = document.querySelector(".wind");
```

Comment:
We don't reassign these variables so these should be better initialized with `const`.